### PR TITLE
Add /EHs for targets for MSVC

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -114,6 +114,12 @@ function(spvtools_default_compile_options TARGET)
     endif()
   endif()
 
+  if (MSVC)
+    # Specify /EHsc for exception handling. This makes using SPIRV-Tools as
+    # dependencies in other projects easier.
+    target_compile_options(${TARGET} PRIVATE /EHsc)
```
Comment (dneto0) [inline CMakeLists.txt]: So what happens if an allocation failure occurs deep inside SPIR-V Tools. Then an exception is thrown.  But most of the SPIRV-Tools API is extern "C".  What happens?

Comment (antiagainst) [inline CMakeLists.txt]: You are right. I haven't thought carefully about the indications of adding `/EHsc` flag. After reading the [spec](https://msdn.microsoft.com/en-us/library/1deeycx5.aspx) a little bit more, I think `/EHs` is more suitable for SPIRV-Tools?

Comment (dneto0) [inline CMakeLists.txt]: I think /EHs is better than /EHsc for SPIRV-Tools.  But does /EHs solve your problem?  If so then update the PR.  :-)

Comment (antiagainst) [inline CMakeLists.txt]: Yes, `/EHs` also works for me. Changed.

**Comment by dneto0** [comment]
Rebased and pushed into master as ad3b08280d69

**Pull request closed without merging**